### ISSUE TITLE
TextField highlight colors on Windows is forced black

### ISSUE THREAD

**Issue by daltzctr** [enisn/UraniumUI#334]
`TextField` (https://enisn-projects.io/docs/en/uranium/latest/themes/material/components/TextField) forcibly sets the `SelectionHighlightColor` to black on Windows. I see no option exposed to change this, and it makes selecting text in a dark theme pretty difficult.

**Comment by daltzctr** [comment]
Relevant text https://github.com/enisn/UraniumUI/blob/a341a54431c24545b08262a54a4a4be030010ddc/src/UraniumUI.Material/Controls/TextField.cs#L62

**Comment by daltzctr** [comment]
Workaround for people:

```
#if WINDOWS
		myTextBox.HandlerChanged += (handler, args) =>
		{
			var textBox = myTextBox.EntryView.Handler.PlatformView as Microsoft.UI.Xaml.Controls.TextBox;
			textBox.SelectionHighlightColor = new Microsoft.UI.Xaml.Media.SolidColorBrush(Microsoft.UI.Colors.Green);
		};
#endif
```

**Comment by enisn** [comment]
Hi, this was a mistake, we'll fix this in the next patch release. Thank you for reporting

**Comment by daltzctr** [comment]
For context, I think the entry has something similar, but unsure.

**Comment by enisn** [comment]
Solved & released in `2.4.6`

**Comment by daltzctr** [comment]
Rad. You're awesome. Thanks.
